source/features/convert-release-to-draft.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import * as api from '../github-helpers/api';
+import GitHubURL from '../github-helpers/github-url';
+import LoadingIcon from '../github-helpers/icon-loading';
+import {getRepoURL, getRepoGQL} from '../github-helpers';
+
+const getReleaseID = async (): Promise<string> => {
+	const tagName = location.pathname.split('/').pop()!;
+	const {repository} = await api.v4(`
+		repository(${getRepoGQL()}) {
+			release(tagName: "${tagName}") {
+				id
+			}
+		}
+	`);
+
+	// => atob(repository.release.id) => "07:Release27300592"
+	return atob(repository.release.id).replace(/.*Release/, '');
+};
+
+async function convertToDraft({delegateTarget: draftButton}: delegate.Event<MouseEvent, HTMLButtonElement>): Promise<void> {
+	draftButton.textContent = 'Converting...';
+	draftButton.append(<LoadingIcon className="ml-2" width={16}/>);
+
+	const releaseID = await getReleaseID();
+	const response = await api.v3(`repos/${getRepoURL()}/releases/${releaseID}`, {
+		method: 'PATCH',
+		body: {
+			draft: true
+		}
+	});
+
+	if (!response.ok) {
+		draftButton.textContent = 'Convert Failed! See console for details';
```

Comment:
This won’t happen. If the response isn’t ok, `api.v3` will throw

Reply:
Uch! your right. I meant to ask you if I should ignore errors and then display that